sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/RecognizePiiEntitiesBatchDocumentsAsync.java

```diff
@@ -37,12 +38,17 @@ public static void main(String[] args) {
 
         // The texts that need be analyzed.
         List<TextDocumentInput> documents = Arrays.asList(
-            new TextDocumentInput("1", "My SSN is 859-98-0987").setLanguage("en"),
+            new TextDocumentInput("1", "My name is Joe and SSN is 859-98-0987").setLanguage("en"),
             new TextDocumentInput("2", "Visa card 4111 1111 1111 1111").setLanguage("en")
         );
 
-        // Show statistics and model version
-        RecognizePiiEntitiesOptions options = new RecognizePiiEntitiesOptions().setIncludeStatistics(true).setModelVersion("latest");
+        // Show statistics, model version, and PII entities that only related to the given Pii entity categories.
+        RecognizePiiEntitiesOptions options = new RecognizePiiEntitiesOptions()
+                                                  .setCategoriesFilter(
+                                                      PiiEntityCategory.USSOCIAL_SECURITY_NUMBER,
+                                                      PiiEntityCategory.CREDIT_CARD_NUMBER)
```

Comment:
idk how useful it is to filter by credit card number in the sample since there are no credit cards in the input document.

Reply:
oops. You are right. I was trying to use the `ABARoutingNumber`

Reply:
Oh sorry again. this is a sample with input text: "Visa card 4111 1111 1111 1111" 
So it should filter the credit card

Reply:
Oh i see it now, missed it the first time. sorry @mssfang

Reply:
No worries at all.  I also missed it when I check it. Thank you for having second eyes on the PR.